app-emulation/wine-vanilla/files/wine-vanilla-6.6-glibc2.34-libresolv.patch

```diff
@@ -0,0 +1,18 @@
+https://bugs.winehq.org/show_bug.cgi?id=51635
+
```

Comment:
same here, we could consider dropping this commit if we only support newer wine

Reply:
fwiw this fix is unrelated to mingw, 6.6 through 6.15 is broken with glibc-2.34 either way, not that I see much worth in that range of versions so could just leave them alone

Reply:
oh ok, since you already did the work lets just keep it